src/ALE/MOM_ALE.F90

```diff
@@ -952,7 +952,7 @@ subroutine ALE_initRegridding(GV, max_depth, param_file, mod, regridCS, dz )
   call initialize_regridding( GV%ke, coordMode, interpScheme, regridCS, &
                               compressibility_fraction=compress_fraction )
 
-  if (coordMode(1:2) == 'Z*') then
+  if (coordMode(1:5) == 'ZSTAR') then
```

Comment:
I see why you're changing this string for z*-coordinates from "Z*" to "ZSTAR" (for 52405299cc6b35d8fba5deb8209a369f60689fe4). Changing a coordinate name is very disruptive now that it's become established (widely used in live models). Normally, when we make such a disruptive change we follow an obsoletion procedure of issuing warnings when using the old name, and then evolve to FATALs after a long time. In this case, I'm thinking we should add a "special case" within MOM_regridding so that both "Z*" and "ZSTAR" work for the model coordinate but not for diagnostic coordinates for which only "ZSTAR" will be used, i.e. keep "Z*" alive in addition to transitioning to "ZSTAR"?